.github/workflows/snyk.yml

```
@@ -2,47 +2,75 @@ name: Snyk Security Scan
-
-      - name: Run Snyk Scan (auth-service)
+        
+      - name: Run Snyk to check for vulnerabilities
         uses: snyk/actions/node@master
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
-          args: test
+          args: --severity-threshold=high
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)